src/definition/DefinitionExecution.js

```diff
@@ -349,7 +349,8 @@ export default function DefinitionExecution(definition) {
     logger.debug(`<${executionId} (${id})>`, reference ? `${messageType} <${delegateMessage.id}>` : `anonymous ${messageType}`, `event received from <${content.parent.id}.${content.id}>. Delegating.`);
 
     getApi().sendApiMessage(messageType, {
-      message,
+      message: message,
+      originalMessage: content.message,
```

Comment:
Had to add a way to expose the original message content from the link throw - in the case of a signal event, there's a separate signal element that can be resolved, but this doesn't exist for link events.